documentation/examples/prometheus.yml

```diff
@@ -27,3 +27,9 @@ scrape_configs:
 
     static_configs:
     - targets: ['localhost:9090']
+
+remote_write:
+  - url: http://localhost:10902/
+    remote_timeout: 30s
+    headers:
+      karesz: kutya
```

Comment:
This file should work out of the box, please do not add this to the example.

Reply:
Sorry, just a mistake! :)